src/browserWindow.ts

```diff
@@ -46,6 +48,39 @@ export class BrowserWindow {
         this.trackerObject.outputDoc = undefined;
       }
     });
+
+    this.webViewPanel.webview.onDidReceiveMessage(
+      (message) => {
+        switch (message.command) {
+          case "clickOnTag_A": {
+            const uri = Uri.parse(message.link);
+            switch (uri.scheme) {
+              case "file":
+                {
+                  const filepath = path.join(filePath, message.link);
```

Comment:
can you check to see if it is an absolute path first, and only join for relative paths?

Reply:
Took care of this.